### PR TITLE
minor tweaks to readme

### PR DESCRIPTION
**Pull request by benbrown**
This addresses some issues @cleemullins pointed out in my copy of this readme file. This brings them into sync.

### CODE REVIEW

File: javascript_ts/2.echobot-with-counter/README.MD [hunk 1]
```diff
@@ -1,15 +1,15 @@
-This sample shows how to create a simple Echo bot with state. The bot keeps simple counter step that increased with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify). 
+This sample shows how to create a simple echo bot with state. The bot keeps simple counter step that increased with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify).
```
Comment (cleemullins) [inline javascript_ts/2.echobot-with-counter/README.MD]: grammar: "increased" should be "increases".

File: javascript_ts/2.echobot-with-counter/README.MD [hunk 2]
```diff
@@ -1,15 +1,15 @@
-This sample shows how to create a simple Echo bot with state. The bot keeps simple counter step that increased with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify). 
+This sample shows how to create a simple echo bot with state. The bot keeps simple counter step that increases with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify).
```
Comment (stevengum) [inline javascript_ts/2.echobot-with-counter/README.MD]: Sorry I'm coming to this late, but can you change the second sentence to sound a little more like the following?

> The bot keeps _a simple step counter_ that increases with each user message.

The original sentence doesn't sound great when spoken aloud. And while I think I understand what is being conveyed, what do we mean by **_step_**?

**Comment by cleemullins** [comment]
@benbrown Once @stevengum feedback is checked in, I think we can merge this.

File: javascript_ts/2.echobot-with-counter/README.MD (hunk 2 above)
Comment (benbrown) [reply javascript_ts/2.echobot-with-counter/README.MD]: I rewrote this sentence slightly to "The bot maintains a simple counter that increases with each message from the user."

**Comment by benbrown** [comment]
merged master, resolved conflicts, should be ready to go once @stevengum signs off

**Comment by cleemullins** [comment]
@benbrown I've pinged @stevengum and he's looking now. Hopefully we can close this out shortly!

File: javascript_ts/2.echobot-with-counter/README.MD [hunk 3]
```diff
@@ -1,9 +1,9 @@
-This sample shows how to create a simple Echo bot with state. The bot keeps simple counter step that increased with each user message. This bot example uses [`restify`](https://www.npmjs.com/package/restify). 
+This sample shows how to create a simple echo bot with state. The bot maintains a simple counter that increases with each message from the user. This bot example uses [`restify`](https://www.npmjs.com/package/restify). 
 
 # To try this sample
 - Clone the repository
     ```bash
-    git clone https://github.com/Microsoft/botbuilder-js.git
+    git clone https://github.com/microsoft/botbuilder-js.git
```
Comment (stevengum) [inline javascript_ts/2.echobot-with-counter/README.MD]: This is pointing to the wrong repository, should be `git clone https://github.com/microsoft/botbuilder-samples.git`

File: javascript_ts/2.echobot-with-counter/README.MD [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
     ```
 - In a terminal, navigate to samples\2.echobot-with-counter
```
Comment (stevengum) [inline javascript_ts/2.echobot-with-counter/README.MD]: This should also be "javascript_nodejs/2.echobot-with-counter"

File: javascript_ts/2.echobot-with-counter/src/index.ts [hunk 5]
```diff
@@ -44,30 +44,30 @@ const adapter = new BotFrameworkAdapter({
...
 
-// Define state store for your bot. See https://aka.ms/about-bot-state to learn more about bots memory service
-// A bot requires a sate store to priciest it dialog and user state between messages
+// Define a state store for your bot. See https://aka.ms/about-bot-state to learn more about using MemoryStorage.
+// A bot requires a state store to persist the dialog and user state between messages.
 const memoryStorage = new MemoryStorage();
 // CAUTION: The Memory Storage used here is for local bot debugging only. When the bot
 // is restarted, anything stored in memory will be gone. 
-// For production bots use the Azure CosmosDB storage, Azure Blob, or Azure Table storage provides. 
+// For production bots use the Azure CosmosDB storage, Azure Blob storage providers.
```
Comment (stevengum) [inline javascript_ts/2.echobot-with-counter/src/index.ts]: Should be "Cosmos DB"

File: javascript_ts/2.echobot-with-counter/src/index.ts [hunk 6]
```diff
@@ -44,30 +44,30 @@ const adapter = new BotFrameworkAdapter({
...
+// Define a state store for your bot. See https://aka.ms/about-bot-state to learn more about using MemoryStorage.
+// A bot requires a state store to persist the dialog and user state between messages.
 const memoryStorage = new MemoryStorage();
 // CAUTION: The Memory Storage used here is for local bot debugging only. When the bot
 // is restarted, anything stored in memory will be gone. 
-// For production bots use the Azure CosmosDB storage, Azure Blob, or Azure Table storage provides. 
+// For production bots use the Azure CosmosDB storage, Azure Blob storage providers. 
 // const { CosmosDbStorage } = require('botbuilder-azure');
-// const STORAGE_CONFIGURATION = 'cosmosDB'; // this is the name of the cosmos DB configuration in your .bot file
+// const STORAGE_CONFIGURATION = 'cosmosDB'; // this is the name of the CosmosDB configuration in your .bot file
```
Comment (stevengum) [inline javascript_ts/2.echobot-with-counter/src/index.ts]: "Cosmos DB"

**Pull request merged**